packages/material/example/index.html

```diff
@@ -8,6 +8,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="assets/native-shim.js"></script>
 
+  <!--- Material Icons -->
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
I already included the dependency to https://github.com/MODX-Club/material-ui-icons, I would suggest to use it instead of the cdn

Reply:
is used by the framework, so we can't use the dependency

Reply:
@eneufeld I don't have the "ignore" button, so can you please click ignore or else remove the request?

Reply:
Hi, 
I just found this: 
> If you dont want to use icon font, or you are already use material-ui-icons you can pass any icon to the components with the following props
leftArrowIcon - arrow left for datepicker
rightArrowIcon - arrow right for datepicker
dateRangeIcon - date tab icon for datetimepicker
timeIcon - time tab icon for datetimepicker
keyboardIcon - icon for keyboard end adornmentic

from the readme: https://github.com/dmtrKovalenko/material-ui-pickers

Reply:
cool, thank you 😄 is changed now